Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7
+
+#declare envs
+ARG SECRET_KEY
+ARG DB_NAME
+ARG DB_USER
+ARG DB_PASSWORD
+ARG DB_HOST
+ARG DB_PORT
+
+#load envs
+ENV SECRET_KEY=${SECRET_KEY} \
```

Comment:
Do you need to declare these here? Seeing as they are used in the compose file, which also references them?

Reply:
These are declared here such that the respective values can be injected by whoever is running the image without the need to create a .env file.

Reply:
Propose to declare all `ENV variables` that will be needed here, and have them defined in the `.env` file.

Reply:
> Propose to declare all `ENV variables` that will be needed here, and have them defined in the `.env` file.

@wakibi could you please elaborate on this more